core/src/main/scala/com/databricks/labs/remorph/parsers/tsql/TSqlExpressionBuilder.scala

```diff
@@ -1,20 +1,53 @@
 package com.databricks.labs.remorph.parsers.tsql
 
-import com.databricks.labs.remorph.parsers.intermediate.{Column, Expression}
+import com.databricks.labs.remorph.parsers.{IncompleteParser, ParserCommon, intermediate => ir}
+import com.databricks.labs.remorph.parsers.intermediate.{Column, Expression, Literal}
+import org.antlr.v4.runtime.tree.TerminalNode
 import com.databricks.labs.remorph.parsers.tsql.TSqlParser._
 
-class TSqlExpressionBuilder extends TSqlParserBaseVisitor[Expression] {
+class TSqlExpressionBuilder
+    extends TSqlParserBaseVisitor[Expression]
+    with ParserCommon
+    with IncompleteParser[ir.Expression] {
+
+  protected override def wrapUnresolvedInput(unparsedInput: String): ir.UnresolvedExpression =
+    ir.UnresolvedExpression(unparsedInput)
 
   override def visitSelect_list_elem(ctx: Select_list_elemContext): Expression = ctx.expression_elem().accept(this)
 
   override def visitExpression(ctx: ExpressionContext): Expression = {
     ctx match {
       case c if c.full_column_name() != null => c.full_column_name().accept(this)
+      case c if c.primitive_expression() != null => c.primitive_expression().accept(this)
     }
   }
 
   override def visitFull_column_name(ctx: Full_column_nameContext): Expression = {
     val columnName = ctx.id_.getText
     Column(columnName)
   }
+
+  override def visitPrimitive_constant(ctx: Primitive_constantContext): Expression = {
+
+    ctx match {
```

Comment:
From our scala style:

For method whose entire body is a pattern match expression, put the match on the same line as the method declaration if possible to reduce one level of indentation.

Reply:
Please read this (whole) document again: https://github.com/databricks/scala-style-guide?tab=readme-ov-file#pattern-matching